packages/html/stories/AutoLayout.stories.ts

```
@@ -152,40 +181,39 @@ const Template = ({ label, ...args }) => {
     overlay.cursor = 'hand';
 
     // Installs a handler for clicks on the overlay
-    overlay.addListener(InternalEvent.CLICK, (sender, evt2) => {
+    overlay.addListener(InternalEvent.CLICK, (_name: string, _funct: Function) => {
```

Biome (1.9.4)

[error] Don't use 'Function' as a type.

Prefer explicitly define the function shape. This type accepts any function-like value, which can be a common source of bugs.

(lint/complexity/noBannedTypes)